.github/workflows/cypress.yml

```diff
@@ -0,0 +1,19 @@
+name: cypress
```

Comment:
Perhaps use a functional name instead, to make clear from the `name` (and the filename) what this workflow is about (that may not be instantly clear with the name 'cypress' - you need some knowledge to understand this)? For example: 'End-to-end testing'?